MapboxVision/Services/MotionManager.swift

```diff
@@ -24,8 +24,9 @@ final class MotionManager {
         motion.showsDeviceMovementDisplay = true
         
         motion.startDeviceMotionUpdates(using: referenceFrame, to: queue) { [weak self] (data, error) in
-            guard let data = data else { return }
-            self?.handler?(data)
+            guard let data = data, let referenceFrame = self?.referenceFrame else { return }
```

Comment:
Consider taking `referenceFrame` from `motion.attitudeReferenceFrame` as an actual utilized value. We don't need to store it as a property then.

Reply:
done